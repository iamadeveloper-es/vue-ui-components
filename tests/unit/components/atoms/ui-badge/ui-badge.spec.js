import { createLocalVue, shallowMount } from "@vue/test-utils";
import UiBadge from '@/components/atoms/ui-badge/index.vue';

describe('Component UiBadge', () => {
    const localVue = createLocalVue();

    const component = shallowMount(UiBadge, {
        localVue
    });

    beforeEach(() => {
        jest.clearAllMocks();
    });

    describe('... Definition', () => {

        test('The name of the component is ui-badge', () => {
            expect(UiBadge.name).toEqual('ui-badge');
        });

        test('Match snapshot', () => {
            expect(component.element).toMatchSnapshot();
        });
    });
});