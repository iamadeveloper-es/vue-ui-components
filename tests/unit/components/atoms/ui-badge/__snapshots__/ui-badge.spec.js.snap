// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component UiBadge ... Definition Match snapshot 1`] = `
<div
  class="ui-badge"
>
  <!---->
</div>
`;
